rpm/SPECS/docker-ce.spec

```diff
@@ -0,0 +1,51 @@
+%global debug_package %{nil}
+
+Name: docker-ce
+Version: %{_version}
+Release: %{_release}%{?dist}
+Epoch: %{getenv:EPOCH}
+Summary: The open-source application container engine
+Group: Tools/Docker
+License: ASL 2.0
+URL: https://www.docker.com
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+Requires: docker-ce-cli
+
+# conflicting packages
+Conflicts: docker
+Conflicts: docker-io
+Conflicts: docker-engine-cs
+Conflicts: docker-ee
+
+# Obsolete packages
+Obsoletes: docker-ce-selinux
+Obsoletes: docker-engine-selinux
+Obsoletes: docker-engine
+
+%description
+Docker is an open source project to build, ship and run any application as a
+lightweight container.
+
+Docker containers are both hardware-agnostic and platform-agnostic. This means
+they can run anywhere, from your laptop to the largest EC2 compute instance and
```

Comment:
I feel like we have a reworded version of this somewhere. 
I could be mistaken, but I think there are things like the largest EC2 instance now says something like largest cloud instance.

Reply:
We probably did, that can be part of a follow up PR by the docs team to align this with our current EE packaging. This PR is meant to get RPM packaging up so we can start testing it with our internal pipelines.

Reply:
> I could be mistaken, but I think there are things like the largest EC2 instance now says something like largest cloud instance.

As a data point, I noticed that specific wording change ("largest EC2 instance" → "largest cloud instance") as well when looking at diffs between .spec file versions a few days ago.  From my point of view, it definitely makes sense. :smile: